package.json

```diff
@@ -26,8 +26,12 @@
         "test:performance:best:ci": "nx test:best:ci @lwc/perf-benchmarks",
         "release:version": "./scripts/release/version.js"
     },
+    "//": {
+        "@babel/helper-explode-assignable-expression": "required for rollup-plugin-compat"
+    },
     "devDependencies": {
         "@babel/core": "^7.22.5",
+        "@babel/helper-explode-assignable-expression": "^7.18.6",
```

Comment:
Not sure why this happened, but compat mode isn't long for this world, so we'll remove this soon anyway